src/System.Data.Common/src/System/Data/IDbTransaction.cs

```diff
@@ -0,0 +1,24 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+
+namespace System.Data{
```

Comment:
I will change it. I had copied over the code from .Net 4.5. 
Thanks for the feedback